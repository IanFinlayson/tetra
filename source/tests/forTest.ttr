def main():
  l = [1...10]
  m = [1,1,2,3,5,8,13,21,34]
  d = {1:"one", 2:"two", 3: "three", 4: "four", 5: "five"}


  for num in l:
    print("num = ")
    print(num)
    print("\n")

  print("\n")
  for fibnum in m:
    print("fibnum = ")
    print(fibnum)
    print("\n")
    print("fibnum^2 = ")
    print(fibnum*fibnum)
    print("\n")

  print("\n")
  for key in d:
    print("key = ")
    print(key)
    print("\n")
    print("val = ")
    print(d[key])
    print("\n")

    
