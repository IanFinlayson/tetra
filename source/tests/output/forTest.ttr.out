num = 1
num = 2
num = 3
num = 4
num = 5
num = 6
num = 7
num = 8
num = 9
num = 10

fibnum = 1
fibnum^2 = 1
fibnum = 1
fibnum^2 = 1
fibnum = 2
fibnum^2 = 4
fibnum = 3
fibnum^2 = 9
fibnum = 5
fibnum^2 = 25
fibnum = 8
fibnum^2 = 64
fibnum = 13
fibnum^2 = 169
fibnum = 21
fibnum^2 = 441
fibnum = 34
fibnum^2 = 1156

key = 1
val = one
key = 2
val = two
key = 3
val = three
key = 4
val = four
key = 5
val = five

